extensions/lifecycle/objectProcessor/LifecycleObjectProcessor.js

```diff
@@ -47,15 +47,17 @@ class LifecycleObjectProcessor extends EventEmitter {
      * @param {Number} s3Config.port - s3 endpoint port
      * @param {String} [transport="http"] - transport method ("http"
      *  or "https")
+     * @param {Object} gcConfig - gc congifuration object
      */
-    constructor(zkConfig, kafkaConfig, lcConfig, s3Config, transport = 'http') {
+    constructor(zkConfig, kafkaConfig, lcConfig, s3Config, transport = 'http', gcConfig) {
```

Comment:
gcConfig should probably be added before the transport field : otherwise we may have issues using the default value for that field (and it seems more consistent to keep the config fields together)